docker_deploy/docker.go

```diff
@@ -40,7 +40,7 @@ func init() {
 }
 
 func NewDockerSensor(ctx context.Context, deps resource.Dependencies, conf resource.Config, logger logging.Logger) (sensor.Sensor, error) {
-	logger.Info("Starting Docker Manager Module v0.0.3")
+	logger.Info("Starting Docker Manager Module...")
```

Comment:
ya, lets remove hardcoded version values XD

Reply:
LOL

Reply:
Ahh yes version "..."

Reply:
I feel very strongly that the module should print it's version on startup for diagnostic purposes. I've since cleaned up some stuff, take a look at https://github.com/viam-soleng/viam-modbus/blob/main/utils/utils.go for a new and improved example.

Reply:
at some point i'll update the build stuff to do it for us automatically